doc/src/devdocs/llvm.md

```diff
@@ -94,10 +94,10 @@ process (e.g. because it creates unserializable state). However, the resulting
 It is also possible to dump an LLVM IR module for just one Julia function,
 using:
 ```julia
-f, T = +, Tuple{Int,Int} # Substitute your function of interest here
+fun, T = +, Tuple{Int,Int} # Substitute your function of interest here
 optimize = false
-open("plus.ll", "w") do f
-    println(f, Base._dump_function(f, T, false, false, false, true, :att, optimize))
+open("plus.ll", "w") do file
+    println(file, Base._dump_function(fun, T, false, false, false, true, :att, optimize, :none, params))
```

Comment:
`params` isn't defined here. This example should be copy-pastable ideally (can be made so by adding default argument values to `_dump_function` if necessary).

Reply:
That was a copy-paste accident, sorry. Should be good now, and ironically copy-pastable

Reply:
Ok, `params` already has a default, this was just there from the code I took from somewhere else for some reason